dev-packages/node-integration-tests/utils/runner.ts

```diff
@@ -565,3 +634,36 @@ function expectLog(item: SerializedLogContainer, expected: ExpectedLogContainer)
     assertSentryLogContainer(item, expected);
   }
 }
+
+/**
+ * Converts ESM import statements to CommonJS require statements
+ * @param content The content of an ESM file
+ * @returns The content with require statements instead of imports
+ */
+function convertEsmToCjs(content: string): string {
```

Comment:
l: not sure if an issue today but should we handle dynamic imports? I don't think they're covered by the conversion cases but might have missed it

Reply:
we do not! but this also does not directly convert to cjs well, I suppose - I'd say for cases where we want to test this, it's probably best to still do this manually (which can still be done the same as before!)